README.md

```diff
@@ -1,7 +1,331 @@
+```csharp
+PermissionIdentity member1 = new PermissionIdentity(@"acme\member1", PermissionIdentityType.User);
+PermissionIdentity member2 = new PermissionIdentity(@"acme\member2", PermissionIdentityType.User);
+
+PermissionIdentityBody bodyMember1 = new PermissionIdentityBody(member1);
+bodyMember1.Mappings.Add(new PermissionIdentity("member1@acme.com", PermissionIdentityType.User));
+
+PermissionIdentityBody bodyMember2 = new PermissionIdentityBody(member2);
+bodyMember2.Mappings.Add(new PermissionIdentity("member2@acme.com", PermissionIdentityType.User));
+
+List<PermissionIdentityBody> mappingsMembersToAddOrUpdate = new List<PermissionIdentityBody> {
+    bodyMember1,
+    bodyMember2
+};
+
+List<PermissionIdentityBody> membersToAddOrUpdate = new List<PermissionIdentityBody> {
+    new PermissionIdentityBody(new PermissionIdentity(@"acme\team2", PermissionIdentityType.Group)) {
+        Members = new List<PermissionIdentity> {
+            member1,
+            member2
+        }
+    }
+};
+
+// Push user mappings and group members in one batched operation.
+client.PermissionManager.AddOrUpdateIdentities(expandedProviderId, null, mappingsMembersToAddOrUpdate.Concat(membersToAddOrUpdate).ToList());
+```
+
+### Disable a single identity
+You can easily disable an identity. For more information, visit [Disabling a Single Security Identity](https://docs.coveo.com/en/84/cloud-v2-developers/disabling-a-single-security-identity).
+```csharp
+client.PermissionManager.DeleteIdentity(expandedProviderId, new PermissionIdentity(@"acme\johndoe", PermissionIdentityType.User));
+```
+
+### Disable identities in batch
+```csharp
+IList<PermissionIdentity> identitiesToDelete = new List<PermissionIdentity> {
+    new PermissionIdentity(@"acme\member1", PermissionIdentityType.User),
+    new PermissionIdentity(@"acme\member2", PermissionIdentityType.User)
+};
+
+client.PermissionManager.DeleteIdentities(expandedProviderId, null, identitiesToDelete);
+```
+
+### Disable identities older than a specific ordering ID
+Same as the documents, you can disable identities that have an ordering ID smaller than the one you provide. For more information, visit [Disabling Old Security Identities](https://docs.coveo.com/en/33/cloud-v2-developers/disabling-old-security-identities).
+```csharp
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, 100, new PermissionIdentityBody(new PermissionIdentity(@"acme\tobedeleted3", PermissionIdentityType.User)));
+client.PermissionManager.AddOrUpdateIdentity(expandedProviderId, 200, new PermissionIdentityBody(new PermissionIdentity(@"acme\tobedeleted4", PermissionIdentityType.User)));
+
+// Wait a little bit, then disable all identities with an ordering ID less than 300.
+client.PermissionManager.DeleteIdentitiesOlderThan(expandedProviderId, 300);
+```
+
+**Good to know:**
+* As for `DeleteDocumentsOlderThan`, there is a processing delay. However, it is not configurable for this call.
```

Comment:
Can we refer to the doc to explain what that delay is. It's not easy to understand what that means when you are not from Coveo, and even then ;)

Reply:
yes jlai rajouté, je l'avais deja mis dans le DeleteDocumentsOlderThan mais bon jvais le rajouter :P

Reply:
```suggestion
* As for `DeleteDocumentsOlderThan`, there is a processing delay. However, it is not configurable for this call. For more information about processing delay, visit [QueueDelay](https://docs.coveo.com/en/131/cloud-v2-developers/deleting-old-items-in-a-push-source). 
```